### PR TITLE
Clarify that workspace admins cannot manage secret teams

### PR DESCRIPTION
**Pull request by JarrettSpiker**
### What
Clarify that workspace admin permissions do not confer the ability to manage the workspace settings of "secret" teams within their organizations

### Why
There was recently some (IMO justified) [confusion around this](https://app.asana.com/0/157256879238414/1202071425828049/f). This is actually already called out in the [Team Visibility](https://www.terraform.io/cloud-docs/users-teams-organizations/teams#team-visibility) section, but adding a note to the workspace permissions side to increase discoverability

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] (n/a) Redirects have been added for moved, renamed, or deleted pages.
- [x] (n/a) Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] (n/a) Pages with related content are updated and link to this content when appropriate.
- [x] (n/a) New pages have metadata (page name and description) at the top.
- [x] (n/a) New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] (n/a) New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] (n/a) UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: content/cloud-docs/users-teams-organizations/permissions.mdx [hunk 1]
```diff
@@ -100,10 +100,12 @@ Workspace admins have the following permissions on their workspace:
...
 - Read and write state versions
 - Read and write workspace settings (workspace admins only) — this includes general settings, notification configurations, run triggers, and more.
-- Set or remove workspace permissions for any team (workspace admins only)
+- Set or remove workspace permissions for visible teams (workspace admins only)
 - Delete workspace (workspace admins only)
 - Manage workspace run tasks
 
+Workspace admins do not implicitly have access to view all teams within their organization. This means that workspace admins are not able to manage workspace permissions for teams which are ["Secret"](/cloud-docs/users-teams-organizations/teams#team-visibility), unless they are also organiztion owners.
+
 See [General Workspace Permissions][] above for details about specific permissions.
```
Comment (laurapacilio) [inline content/cloud-docs/users-teams-organizations/permissions.mdx:109]:
```suggestion
Workspace admins cannot view teams that are are ["Secret"](/cloud-docs/users-teams-organizations/teams#team-visibility), unless they are also organization owners.

Refer to [General Workspace Permissions][] for details about specific permissions.
```

**Review by laurapacilio — CHANGES_REQUESTED** [review]
I actually think it might be more helpful to move this sentence about how if you're a workspace admin, you can't always see secret teams to be RIGHT after the line about visible teams. I feel like when I read it I was like "what?" and then I understood when I saw your explanation! 

Can we try that and see how we feel about it? Thank you for making this change!

File: content/cloud-docs/users-teams-organizations/permissions.mdx [hunk 2]
```diff
@@ -100,11 +100,12 @@ Workspace admins have the following permissions on their workspace:
 - Read and write variables
 - Read and write state versions
 - Read and write workspace settings (workspace admins only) — this includes general settings, notification configurations, run triggers, and more.
-- Set or remove workspace permissions for any team (workspace admins only)
+- Set or remove workspace permissions for visible teams (workspace admins only)
+  - Workspace admins cannot view or manage teams that are are ["Secret"](/cloud-docs/users-teams-organizations/teams#team-visibility), unless they are also organization owners.
```
Comment (laurapacilio) [inline content/cloud-docs/users-teams-organizations/permissions.mdx:104]: Maybe this?  Also this is kind of related but also unrelated, isn't "workspace admins only" implied by the "Workspace admins have the following permissions" statement above all of these bullets? If so I'm wondering why we need it in some of these....

```suggestion
- Set or remove workspace permissions for visible teams (workspace admins only). Workspace admins cannot view or manage teams that are are ["Secret"](/cloud-docs/users-teams-organizations/teams#team-visibility), unless they are also organization owners.
```

Comment (JarrettSpiker) [reply content/cloud-docs/users-teams-organizations/permissions.mdx]: > Also this is kind of related but also unrelated, isn't "workspace admins only" implied by the "Workspace admins have the following permissions" statement above all of these bullets? If so I'm wondering why we need it in some of these....

There are a bunch of workspace permissions which can be [assigned to any team](https://www.terraform.io/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions), but you need to be a workspace admin to do some things (like deleting the workspace, or changing the permissions of. other teams)

So I think it is useful to call those out ("hey, you need to be an admin to do these things")...maybe we could change the list to be things which **only** workspace admins do, as opposed to **all** the things workspace admins do? Something like "Workspace admins have all permissions described in [General Permissions](https://www.terraform.io/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions), as well as the ability to..."?

Comment (laurapacilio) [reply content/cloud-docs/users-teams-organizations/permissions.mdx]: YESSSS I actually think I like that approach you just mentioned. Sorry to balloon this PR into more than you anticipated, but I would be interested in seeing how that looks :)

Comment (JarrettSpiker) [reply content/cloud-docs/users-teams-organizations/permissions.mdx]: Updated as described, to remove the duplication with the above section and focus the permissions here on the stuff exclusive to workspace admins. Let me know what you think @laurapacilio !
Screen Shot 2022-05-19 at 3 14 09 PM

File: content/cloud-docs/users-teams-organizations/permissions.mdx [hunk 3]
```diff
@@ -90,21 +90,11 @@ Much like the owners team has full control over an organization, each workspace
...
-- Read and write variables
-- Read and write state versions
-- Read and write workspace settings (workspace admins only) — this includes general settings, notification configurations, run triggers, and more.
-- Set or remove workspace permissions for any team (workspace admins only)
-- Delete workspace (workspace admins only)
-- Manage workspace run tasks
-
-See [General Workspace Permissions][] above for details about specific permissions.
+- Read and write workspace settings — this includes general settings, notification configurations, run triggers, and more.
+- Set or remove workspace permissions for visible teams. Workspace admins cannot view or manage teams that are are ["Secret"](/cloud-docs/users-teams-organizations/teams#team-visibility), unless they are also organization owners.
```
Comment (laurapacilio) [inline content/cloud-docs/users-teams-organizations/permissions.mdx:96]:
```suggestion
- Set or remove workspace permissions for visible teams. Workspace admins cannot view or manage teams that are are [Secret](/cloud-docs/users-teams-organizations/teams#team-visibility), unless they are also organization owners.
```

File: content/cloud-docs/users-teams-organizations/permissions.mdx [hunk 4]
```diff
@@ -90,21 +90,11 @@ Much like the owners team has full control over an organization, each workspace
 
 Admin permissions include the highest level of general permissions for the workspace. There are also some permissions that are only available to workspace admins, which generally involve changing the workspace's settings or setting access levels for other teams.
 
-Workspace admins have the following permissions on their workspace:
+Workspace admins have all the permissions described in [General Workspace Permissions][], as well as the ability to:
```
Comment (laurapacilio) [inline content/cloud-docs/users-teams-organizations/permissions.mdx:93]:
```suggestion
Workspace admins have all [General Workspace Permissions][], as well as the ability to do the following tasks:
```

File: content/cloud-docs/users-teams-organizations/permissions.mdx [hunk 5]
```diff
@@ -90,21 +90,11 @@ Much like the owners team has full control over an organization, each workspace
...
-- Download Sentinel mocks
-- Read and write variables
-- Read and write state versions
-- Read and write workspace settings (workspace admins only) — this includes general settings, notification configurations, run triggers, and more.
-- Set or remove workspace permissions for any team (workspace admins only)
-- Delete workspace (workspace admins only)
-- Manage workspace run tasks
-
-See [General Workspace Permissions][] above for details about specific permissions.
+- Read and write workspace settings — this includes general settings, notification configurations, run triggers, and more.
```
Comment (laurapacilio) [inline content/cloud-docs/users-teams-organizations/permissions.mdx:95]:
```suggestion
- Read and write workspace settings. This includes general settings, notification configurations, run triggers, and more.
```

**Review by laurapacilio — APPROVED** [review]
Approving so I don't block. A couple of nits but then you're good to 🚀  Thank you! I thin kthis reads much better and makes the page just a teensy bit shorter (which we can all appreciate on long pages like this one!!)

**Comment by JarrettSpiker** [comment]
> Approving so I don't block. A couple of nits but then you're good to 🚀 Thank you! I thin kthis reads much better and makes the page just a teensy bit shorter (which we can all appreciate on long pages like this one!!)

Awesome, thanks for all the help Laura!

**Pull request merged**